app/controllers/collection_resources_controller.rb

```diff
@@ -178,6 +178,29 @@ def create
     end
   end
 
+  def delete_confirmation
+    return render_not_found unless current_user.support?
```

Comment:
Would it make sense to use the `render_not_authorized` method here instead? I know they eventually do the same thing but for readability. Keeping it `render_not_found` below when there is not collection resource.

Reply:
Yup, updated now!